test/functional/site/server.js

```diff
@@ -50,6 +53,31 @@ Server.prototype._setupRoutes = function () {
                 res.sendStatus(404);
             });
     });
+
+    this.app.post('/file-upload', upload.any(), function (req, res) {
+        // NOTE: uploadedFileArrays is an array which includes the file arrays which
+        // were added with the browsers.
+        // uploadedFileArrays = [browserXUploadedFiles, browserYUploadedFiles, ...];

```

Comment:
are we need for a text explanation here?
IMHO it's enough `// NOTE: uploadedFileArrays = [browserXUploadedFiles, browserYUploadedFiles, ...];`

/cc @VasilyStrelyaev